src/helpers/Move.js

```diff
@@ -28,21 +29,130 @@ export const move = (move, position) => {
         let moveColumn = move[0], moveRow = move[1];
 
         if(players[currentPlayer] === 'white'){         
-            var whiteMoveColumnFields =  fields.filter( field => field.column === moveColumn);
-            var whitemoveField = whiteMoveColumnFields.filter(field => field.row === Number(moveRow));
+            let whiteMoveColumnFields =  fields.filter( field => field.column === moveColumn);
+            let whitemoveField = whiteMoveColumnFields.filter(field => field.row === Number(moveRow));
             whitemoveField[0].piece = 'P';
-            var whitecandidateFields = whiteMoveColumnFields.filter( field => field.row < moveRow );
-            var whitesetFreeField = whitecandidateFields.filter( field => field.piece === 'P');
+            let whitecandidateFields = whiteMoveColumnFields.filter( field => field.row < moveRow );
+            let whitesetFreeField = whitecandidateFields.filter( field => field.piece === 'P');
             whitesetFreeField[0].piece = '';      
         } else {
-            var blackMoveColumnFields =  fields.filter( field => field.column === moveColumn);
-            var blackMoveField = blackMoveColumnFields.filter(field => field.row === Number(moveRow));
+            let blackMoveColumnFields =  fields.filter( field => field.column === moveColumn);
+            let blackMoveField = blackMoveColumnFields.filter(field => field.row === Number(moveRow));
             blackMoveField[0].piece = 'p';
-            var blackCandidateFields = blackMoveColumnFields.filter( field => field.row > moveRow );
-            var blackSetFreeField = blackCandidateFields.filter( field => field.piece === 'p');
+            let blackCandidateFields = blackMoveColumnFields.filter( field => field.row > moveRow );
+            let blackSetFreeField = blackCandidateFields.filter( field => field.piece === 'p');
             blackSetFreeField[0].piece = '';
         }
-    } 
+    } else if(move.length === 3){
+
+        let piece = move[0], moveColumn = move[1], moveRow = move[2];
+        let columnInd;
+        switch(moveColumn){
+            case 'a':
+                columnInd = 0;
+                break;
+            case 'b':
+                columnInd = 1;
+                break;
+            case 'c':
```

Comment:
Maybe to have one variable column with mapped values like column = [a: '0', b: '1' ..] and when you wish to call it use
moveColumn[move[1]]